presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailUiModel.kt

```diff
@@ -1,3 +1,3 @@
 package com.architect.coders.mu8.characters.detail
 
-sealed class CharactersDetailUiModel
\ No newline at end of file
+sealed class CharactersDetailUiModel
```

Comment:
What change was done here?

Reply:
sorry i have detekt automatic and i don't realize the change. I'm going to put it back the way it was.